db/migrate/20160729094457_add_columns_to_ansible_role.rb

```diff
@@ -0,0 +1,12 @@
+#:nodoc:
+class AddColumnsToAnsibleRole < ActiveRecord::Migration
+  def up
+    add_column :ansible_roles, :created_at, :datetime
+    add_column :ansible_roles, :updated_at, :datetime
```

Comment:
Add a default value for this - equal to the Time.now or something - otherwise this breaks https://github.com/theforeman/foreman_ansible/pull/25/files#diff-97a0da96e3dffb0eb149878984f699edR17 when calling .updated_at on pre-existing roles (remember there are roles already in the system prior to this PR)